docs/source/reference/iterators.rst

```diff
@@ -17,3 +17,18 @@ Chainer provides some iterators that implement typical strategies to create mini
    chainer.iterators.SerialIterator
    chainer.iterators.MultiprocessIterator
    chainer.iterators.MultithreadIterator
+
+
+Order sampler examples
+======================
+
+An Iterator iterates over a dataset according to an order represented by an 1-D array of indices.
```

Comment:
Sorry this was my mistake but it should be `a 1-D array`.

Reply:
Right. Thanks.